### DISCUSSION TITLE
Naming inconsistency in attributes of chars?

### DISCUSSION THREAD

**Discussion by alexreg** [jsvine/pdfplumber#384] [Q&A]
Is there any reason that the bbox of a char is represented by the attributes (`x0`, `top`, `x1`, `bottom`), and not (`x0`, `y0`, `x1`, `y1`)? I was using the latter for a while and really wondering why my values were wrong.

**Comment by jsvine** [comment]
It's a good question, and one that's probably worth explaining more clearly in the documentation. In the meantime, for the answer and an outline of my thinking on the topics, please see this short discussion: https://github.com/jsvine/pdfplumber/issues/198

**Reply by alexreg** [reply]
Thanks, that's good for the meanwhile. Would definitely be nice to have a succinct explanation in the docs, but that helps for now. (You can also explain the `doctop` attribute there.)

**Reply by jsvine** [reply]
Self-assigned an issue here: https://github.com/jsvine/pdfplumber/issues/389
